languages/julia/concepts/multiple-dispatch/links.json

```diff
@@ -1 +1,14 @@
-[]
+[
+    {
+        "url": "https://giordano.github.io/blog/2017-11-03-rock-paper-scissors/",
+        "description": "\"Rock–paper–scissors game in less than 10 lines of code\", by Mosè Giordano."
+    },
+    {
+        "url": "https://youtu.be/kc9HwsxE1OY",
+        "description": "\"The Unreasonable Effectiveness of Multiple Dispatch\", by Stefan Karpinski at JuliaCon 2019."
+    },
+    {
+        "url": "https://docs.julialang.org/en/v1/manual/methods/#man-methods-orthogonalize-1",
+        "description": "\"Orthogonalize your methods design\", Julia Manual."
+    }
+]
```

Comment:
I don't think we should shorten these names and they will look bad in the proposed UI in #2499. Perhaps we should just have an empty links.json?

Reply:
I don't think it is very important to use whether the links appear in the text or in little buttons, so I think it's fine for us to have the links.json as an empty array.

Maybe we won't get automatic link liveliness checking, but that doesn't matter much (and anyway tooling could extract links from the prose for testing). Do we lose anything else?

Reply:
Yeah I had a lengthy discussion about this on the Slack to figure out what exactly these should be, and how one can shorten them without losing important info. 🤷 Let's remove them for now.

> Maybe we won't get automatic link liveliness checking

There's a CI run that does that for links in markdown files already.